doc/telemetry/kusto_examples.md

```diff
@@ -0,0 +1,99 @@
+Samples for kusto query
+=======================
+### Query for new schema
+CLI telemetry has different schema after version 2.0.28
+[Execute in Web](https://dataexplorer.azure.com/clusters/ddazureclients/databases/AzureCli?query=H4sIAAAAAAAAAwtKLHctS80rKXascs7J5KpRKM9ILUpVKC7IySzRCCjKTylNLglLLSrOzM/TUVDXU9eMNohVsLVVUE+sKi1KTc7JdDBSh+sqyc/MK9HAo9cwVlPBTsFAIb+IsFojsFojC6DpJYnZqQqmAIl8AharAAAA)
```

Comment:
A line break is needed. Otherwise, `Execute in Web` is concatenated to `CLI telemetry has different schema after version 2.0.28`:

![image](https://user-images.githubusercontent.com/4003950/173535599-10dc1f87-9394-47df-9ee1-26c53d4eeddc.png)